apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/ClaimTypeV2.java

```diff
@@ -80,6 +90,14 @@ public String getEntityMbiHashAttribute() {
     return entityMbiHashAttribute;
   }
 
+  public String getEntityStartDateAttribute() {
```

Comment:
If we're now using lombok, can't we just use `@Getter` for this?

Reply:
lombok is only in the bfd-model-rda module, it has not been introduced to bfd-server-war